src/pages/resultsView/enrichments/MiniScatterChart.tsx

```diff
@@ -52,23 +55,39 @@ export default class MiniScatterChart extends React.Component<
                 : null;
     }
 
+    @computed get isGenericAssay() {
+        return !!this.props.onGenericAssayEntityClick;
+    }
+
```

Comment:
Same as `ExpressionEnrichmentsBoxPlot.tsx`, we can use `onGenericAssayEntityClick` to know if this is a plot for Generic Assay data.